storybook/config.js

```diff
@@ -16,8 +16,10 @@ addDecorator( withA11y );
 addDecorator( withKnobs );
 configure(
 	[
-		require.context( './stories/', true, /\/.+\.(js|mdx)$/ ),
+		// StoryShots addon doesn't support MDX files at the moment.
+		// It should ignore the playground in the initial pass as well.
+		process.env.NODE_ENV !== 'test' && require.context( './stories/', true, /\/.+\.(js|mdx)$/ ),
```

Comment:
Could not calling require.context for test NODE_ENV have another undesired effect? Would it make sense to in the test environment call require.context but not include mdx in the regex?

Reply:
The issue is that we are calling it for JS files twice now. See line 20/22.

Reply:
There are two different entry paths defined in line 20:
```js
require.context( './stories/', true, /\/.+\.(js|mdx)$/ )
```
and line 22:
```js
require.context( '../packages/components/src/', true, /\/stories\/.+\.js$/ )
```

> Could remove the call require.context for test NODE_ENV have an undesired effect? Would it make sense to in the test environment call require.context but not include mdx in the regex?

It depends if we want to enable any stories in the `storybook/stories` folder. At the moment it contains MDX file and the playground. We can always split MDX to its own line and let Storyshot run the test again playground. It might work, I didn't check.

Reply:
Oh right, I missed that :sweat_smile:

Reply:
I think it's fine as is for now then.